app/frontend/src/components/MarkdownInput/MarkdownInput.tsx

```diff
@@ -75,6 +80,31 @@ export default function MarkdownInput({
   const fieldOnChange = useRef<typeof field.onChange>(field.onChange);
 
   useEffect(() => {
+    const uploadButton = imageUpload ? document.createElement("button") : null;
```

Comment:
Is adding a custom button to the editor a thing since their v3 editor? That's pretty neat!